microsoft/electionguard-api-python
# Configure poetry to use venv in project

### Description
configure poetry to place venvs in projects by default

The VS Code poetry venv path was set to a specific location on someone's mac.  If we have to use a specific location, it may as well be in the project's path.


### Testing
1. run the make environment script again
2. ensure you get a .venv folder
3. change the python interpreter path to the one in the .venv folder in vs code
4. make build
5. validate vs code picks up all the modules and whatnot
